app/src/main/res/drawable/news_selector.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@drawable/ic_news_list_on" android:state_selected="true"/>
+    <item android:drawable="@drawable/ic_news_list_off"/>
```

Comment:
Please use state_selected="false" for ic_news_list_off. In other cases we will want another state for default situations